conf/example_usm.ini

```diff
@@ -7,6 +7,9 @@
 usm_log_level = DEBUG
 usm_username = <username>
 usm_password = <password>
-usm_web_url = https://example-usm3-server.usmqe.tendrl.org
+usm_web_url = http://example-usm3-server.usmqe.tendrl.org
```

Comment:
Default is http right now. I think it is better if example reflects reality. I know that in future it should be https.